### ISSUE TITLE
Broken default login page

### ISSUE THREAD

**Issue by Perceval62** [hughbris/grav-theme-landed#12]
# What's up ?
Hey, the theme is absolutely gorgeous. I love it!
Small quirk, the user login screen on grav (/login) has a white font on a white background.  
![image](https://user-images.githubusercontent.com/31328738/210157005-c53cef97-c387-449c-9f94-fc1e84b969e5.png)
The Admin login panel is still good though.

# How to replicate 
Install grav on Linode, install the landed theme, go to the /login page.

Have a happy new year 🎉

**Comment by hughbris** [comment]
> Hey, the theme is absolutely gorgeous. I love it!

Belated thanks and I agree, though this praise really should be directed @ajlkn, the theme designer.

> user login screen on grav (/login) has a white font on a white background

Back when I used to put more energy into CSS (before we had great open licensed themes), I remember a rule of thumb I probably picked up from a blog somewhere (or maybe it was WCAG):

> _if you set one colour, set them all_ 

(they probably used American spelling though). It was for precisely this problem.

**TLDR; I don't really think this is for the theme to solve.** There are a number of ways to work around it.

Grav's login plugin sets the background style in its CSS (without setting a foreground colour!) and contains the offending template that uses it.

You could hack/override this template, or add a style in some custom CSS like:

```CSS
#grav-login {
    color: #000; /* or some nice shade from the theme, OR... */
    /* background: initial; */ /* or specify a color */
}
```

This theme has a `custom.css` file you can add extra styles to. This will also work if you use a theme that inherits Landed.

So I think is really for the login plugin to make good. I suggest [adding an issue there](https://github.com/getgrav/grav-plugin-login/issues) but you could be waiting some time for that. In the meantime, you have simple workarounds available.

I am going to mention `custom.css` in the docs, as I think this is an important capability to point out.

Sorry for taking so long to respond.
